change/@fluentui-react-dialog-bbe00f26-9da5-425b-be2b-d6add0d2ff1f.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 12 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27845&commitId=d5c5f3e186d93f55c6cae75c016402c0c371a3e2&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1924 screenshots unchanged, 0 screenshots with different dimensions and 12 screenshots with visible difference.


  <details><summary><b>unknown</b> 12 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Dialog.no actions - Dark Mode.chromium.png|6535| Changed|
|Dialog.no actions - High Contrast.chromium.png|9447| Changed|
|Dialog.no actions - RTL.chromium.png|7263| Changed|
|Dialog.no actions.chromium.png|7256| Changed|
|Dialog.no title & no actions - Dark Mode.chromium.png|6043| Changed|
|Dialog.no title & no actions - High Contrast.chromium.png|8859| Changed|
|Dialog.no title & no actions - RTL.chromium.png|6757| Changed|
|Dialog.no title & no actions.chromium.png|6751| Changed|
|Dialog.title custom action - Dark Mode.chromium.png|6948| Changed|
|Dialog.title custom action - High Contrast.chromium.png|9964| Changed|
|Dialog.title custom action - RTL.chromium.png|7691| Changed|
|Dialog.title custom action.chromium.png|7687| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>